### ISSUE TITLE
cssClasses - how to use, and is it even supported?

### ISSUE THREAD

**Issue by smileBeda** [leongersen/noUiSlider#1279]
[Here](https://refreshless.com/nouislider/more/#section-styling) it is said that `cssClasses` is an option but the [option doc](https://refreshless.com/nouislider/slider-options/) does not mention it, and no matter what I pass to the option, it always ends in `undefined` classes produced in the HTML.

I then found https://github.com/leongersen/noUiSlider/blob/faa3252f84d3e2416ba43ba49b2de91b9112ce80/src/nouislider.ts#L4 and if I pass **all of those classes** like :
```
cssClasses: {
    target: " string",
    base: " string",
    origin: " string",
    handle: " string",
    handleLower: " string",
    handleUpper: " string",
    touchArea: " string",
    horizontal: " string",
    vertical: " string",
    background: " string",
    connect: " string",
    connects: " string",
    ltr: " string",
    rtl: " string",
    textDirectionLtr: " string",
    textDirectionRtl: " string",
    draggable: " string",
    drag: " string",
    tap: " string",
    active: " string",
    tooltip: " string",
    pips: " string",
    pipsHorizontal: " string",
    pipsVertical: " string",
    marker: " string",
    markerHorizontal: " string",
    markerVertical: " string",
    markerNormal: " string",
    markerLarge: " string",
    markerSub: " string",
    value: " string",
    valueHorizontal: " string",
    valueVertical: " string",
    valueNormal: " string",
    valueLarge: " string",
    valueSub: " string",
}
```

well... it produces HTML elements with `noUi- string`, which makes absolutely no sense to me. it seems to _replace_ the class rather than append, and only works if we overwrite them all?
Indeed, I can then also overwrite that with `cssPrefix: ''`, but... that fully defies the purpose of this, IMO.

The final goal is to add classes from Bootstrap - and that just becomes impossible, if we have to either remove the `noUi` prefix just to add something like a bg-danger (thus totally breaking noUI) or append to it, therefore breaking Bootstrap.

Is there any solution to this?

**Comment by leongersen** [comment]
> it seems to replace the class rather than append, and only works if we overwrite them all?

Correct, that is what this option does. This option is for projects that combine and style multiple UI elements into a library. I would not recommend using it in a one-off project.
